creator-node/src/routes/files.js

```diff
@@ -317,6 +317,14 @@ const getDirCID = async (req, res) => {
 }
 
 module.exports = function (app) {
+  app.get('/processing_status', handleResponse(async (req, res) => {
+    const taskType = req.query.taskType
+    const uuid = req.query.uuid
+    const value = await redisClient.get(`${taskType}:::${uuid}`) || '{}'
```

Comment:
lets make this a function in the FileProcessingQueue so it's not declared twice?